### PR TITLE
`Development`: Add documentation for local multiple Artemis instances setup

### PR DESCRIPTION
**Pull request by laurenzfb**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-
### Motivation and Context

The documentation for the multiple Artemis instances setup is missing some information how you can do the setup locally on a single machine (for testing purposes). This PR adds some instructions for this.

### Description
I added a subsection "Running multiple instances locally" to the section "Multiple Artemis Instances".

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### Screenshots

image

### CODE REVIEW

File: docs/admin/setup/distributed.rst [hunk 1]
```diff
@@ -502,3 +502,29 @@ It relays message between instances:
...
+1. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``
+
+2. Create additional run configurations for each instance
+
+You will have to add CLI arguments to each additional run configuration to set the instance ID and the port,
+e.g. ``--server.port=8081 --eureka.instance.instanceId="Artemis:2"``. Also make sure that only one instance has
+the ``scheduling`` profile enabled:
+
+   .. figure:: distributed/run-config.png
+      :align: center
```
Comment (b-fein) [inline docs/admin/setup/distributed.rst:521]: This should probably be indented once to be part of point 2.

File: docs/admin/setup/distributed.rst [hunk 2]
```diff
@@ -502,3 +502,29 @@ It relays message between instances:
...
    .. figure:: distributed/registry.png
       :align: center
+
+
+Running multiple instances locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+1. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``
```
Comment (b-fein) [inline docs/admin/setup/distributed.rst:512]:
```suggestion
1. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``.
```
nitpick: periods missing at the end of full sentences for all five steps.

File: docs/admin/setup/distributed.rst [hunk 3]
```diff
@@ -502,3 +502,29 @@ It relays message between instances:
...
+      :align: center
+
+3. Start the registry service, e.g., by running ``docker compose -f docker/broker-registry.yml up``
+
+4. Start the first instance with the default run configuration (no additional CLI arguments, ``scheduling`` enabled.)
+   and wait until it is registered in the registry
+
+5. Start the remaining instances
+
+You should now be able to see all instances in the registry interface.
```
Comment (b-fein) [inline docs/admin/setup/distributed.rst:530]:
```suggestion
You should now be able to see all instances in the registry interface at ``http://localhost:????``.
```
Might be helpful to add the port here when running as described in step 3 via Docker Compose.

**Review by b-fein — DISMISSED** [review]

**Comment by laurenzfb** [comment]
Thanks for the review @b-fein! Implemented your suggested changes

**Review by b-fein — APPROVED** [review]

**Review by florian-glombik — APPROVED** [review]
Instructions are clear to me 👍

File: docs/admin/setup/distributed.rst [hunk 4]
```diff
@@ -502,3 +502,27 @@ It relays message between instances:
...
    .. figure:: distributed/registry.png
       :align: center
+
+
+Running multiple instances locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+#. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``.
```
Comment (tobias-lippert) [inline docs/admin/setup/distributed.rst:512]: shouldn't this value rather be set in application-local.yml?

Comment (laurenzfb) [reply docs/admin/setup/distributed.rst:512]: You're right. Changed it.

**Review by tobias-lippert — APPROVED** [review]
LGTM

File: docs/admin/setup/distributed.rst [hunk 5]
```diff
@@ -502,3 +502,34 @@ It relays message between instances:
...
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+#. In ``application-local.yml``, add the following configuration:
+
+  .. code:: yaml
+
+    eureka:
+        client:
+            enabled: true
```
Comment (b-fein) [inline docs/admin/setup/distributed.rst:518]:
```suggestion
  .. code:: yaml

     eureka:
         client:
             enabled: true
```
Needs to be indented with one additional space to fix the item numbering.

Currently:
image

Comment (laurenzfb) [reply docs/admin/setup/distributed.rst]: sorry about that. Now fixed

**Review by b-fein — DISMISSED** [review]

**Review by egekurt123 — APPROVED** [review]
Instructions look clear and easy to understand

**Review by JanaNF — CHANGES_REQUESTED** [review]
Instructions are clear. When I copied your text into Grammarly (we get from the TUM free access to the education plan) I found following things you could improve:
1. Missing comma in the sentence (second item): 
> `Also` make sure that...

2. Please consider removing passive usage from the text.
> `This` can `be done` by using different ports and a unique instance ID for each instance.

> 7. Start the first instance with the default run configuration (no additional CLI arguments, scheduling enabled.) and wait until it `is registered` in the registry.

**Review by tobias-lippert — APPROVED** [review]
Reapprove

**Review by JanaNF — APPROVED** [review]

**Review by egekurt123 — APPROVED** [review]

**Review by florian-glombik — APPROVED** [review]

**Review by LenaKahle — APPROVED** [review]
Looks good.

**Pull request merged**